test/AutoDiff/SILOptimizer/differentiation_diagnostics.swift

```diff
@@ -741,7 +741,7 @@ struct TF_675 : Differentiable {
 let _: @differentiable(reverse) (Float) -> Float = TF_675().method
 
 // TF-918: Test parameter subset thunk + partially-applied original function.
```

Comment:
I can no longer find TF-918... Is there a GitHub issue for it?

Reply:
I'm afraid all TF issues were gone long ago. And if they were not converted to normal Swift JIRA issues, then they cannot be found anywhere...

@shahmishal will you please confirm?

Reply:
We should probably go over all `TF-` bugs in comments and remove them.

Reply:
And rewrite to present issues when the issue was migrated (there are few of them IIRC). Some of the TF's refer to some design decisions made in the past. It would be good to document them somehow while there are some memories on them :) I'll check what are the most important / interesting ones